--- a/node.c
+++ b/node.c
@@ -301,6 +301,7 @@
   node_log(LOGLVL_LOGIN, "%s @ %s logged in", User.call, User.ul_name);
 #ifdef HAVEMOTD
   if (User.ul_type == AF_NETROM) {
+	  axio_printf(NodeIo, "Fit like eh day Netrommie. \n");
     /*		axio_printf(NodeIo, "%s} Welcome.\n", NodeId);  */
   } else 
     if ((User.ul_type == AF_INET) || (User.ul_type == AF_INET6)) {
@@ -362,6 +363,7 @@
 	  else if ((User.ul_type != AF_NETROM) && (User.ul_type != AF_ROSE)) {
 	    node_prompt();
 	  } else if ((User.ul_type == AF_NETROM) || (User.ul_type == AF_ROSE)) {
+		  node_prompt();
 	    axio_printf(NodeIo,"\n");
 	  }
 	}
