--- a/node.c
+++ b/node.c
@@ -301,6 +301,11 @@
   node_log(LOGLVL_LOGIN, "%s @ %s logged in", User.call, User.ul_name);
 #ifdef HAVEMOTD
   if (User.ul_type == AF_NETROM) {
+	node_msg("%s - Welcome to %s", VERSION, NodeId); 
+		if ((fp = fopen(HAVEMOTD, "r")) != NULL) {
+		while (fgets(buf, 256, fp) != NULL) axio_puts(buf,NodeIo);
+		axio_puts ("\n",NodeIo);
+	}
     /*		axio_printf(NodeIo, "%s} Welcome.\n", NodeId);  */
   } else 
     if ((User.ul_type == AF_INET) || (User.ul_type == AF_INET6)) {
@@ -365,6 +370,8 @@
 	    axio_printf(NodeIo,"\n");
 	  }
 	}
+    } else {
+	    node_prompt();
     }
     if (check_perms(PERM_ANSI, 0L) != -1) {
       /* Not needed from what I see so far.        */
