--- a/configure
+++ b/configure
@@ -74,14 +74,15 @@
 _sleep 1
 echo -n "Checking for the existence of the Zlib headers... "
 ZLIB=""
-HAVEZLIB="#undef HAVE_ZLIB"
+HAVEZLIB="#undef HAVE_ZLIB_H"
 for zlibdir in /usr/include /usr/local/include
 do
 	if [ -f $zlibdir/zlib.h ]
 	then
-		echo $zlibdir/zlib.h
-		ZLIB="-lz"
-		HAVEZLIB="#define HAVE_ZLIB	1"
+		echo "skipping zlib until it works right..."
+		# echo $zlibdir/zlib.h
+		# ZLIB="-lz"
+		# HAVEZLIB="#define HAVE_ZLIB_H	1"
 	fi
 done
 if [ -z "$ZLIB" ]
